File: LeetCodeSolutions/1480-RunningSum/RunningSum.c

/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* runningSum(int* nums, int numsSize, int* returnSize) {
    *returnSize = numsSize;
    int newnum = 0;
    int* res = malloc(numsSize * sizeof(int));
    for(int i = 0; i<numsSize; i++){
        newnum = nums[i] + newnum;
        res[i] = newnum;
    }
    return res;
    
}

File: LeetCodeSolutions/1480-RunningSum/RunningSum.py

from typing import List


class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        newnum = 0
        res = []
        for i in range(len(nums)):
            newnum = nums[i] + newnum
            res.append(newnum)
        return res
   